store/sqlstore/upgrade.go

```diff
@@ -531,6 +531,10 @@ func UpgradeDatabaseToVersion56(sqlStore SqlStore) {
 			sqlStore.RemoveIndexIfExists("idx_users_firstname_lower", "lower(FirstName)")
 			sqlStore.RemoveIndexIfExists("idx_users_lastname_lower", "lower(LastName)")
 		}
+
+		sqlStore.CreateIndexIfNotExists("idx_groupmembers_create_at", "GroupMembers", "CreateAt")
```

Comment:
We don't usually have `CreateIndexIfNotExists` within the `upgrade.go` -- rather, this is done within `NewSqlSupplier`. 2/5 against adopting a new pattern.